src/main/java/com/java_essential/safonov/homework/lesson7/Worker/Main.java

```diff
@@ -0,0 +1,61 @@
+package com.java_essential.safonov.homework.lesson7.Worker;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.text.ParseException;
+import java.time.Year;
+/*
+Задание 2
+Требуется: Описать класс с именем Worker, содержащую следующие поля:
+ фамилия и инициалы работника;
+ название занимаемой должности;
+ год поступления на работу.
+Написать программу, выполняющую следующие действия:
+ввод с клавиатуры данных в массив, состоящий из пяти элементов типа Worker (записи должны быть упорядочены по алфавиту);
+если значение года введено не в соответствующем формате выдает исключение.
+вывод на экран фамилии работника, стаж работы которого превышает введенное значение.
+ */
+
+public class Main {
+    public static void main(String[] args) throws IOException, ParseException {
```

Comment:
смотрите примеры по уроку 7
```suggestion
    public static void main(String[] args) {
```